listenbrainz/webserver/views/index.py

```diff
@@ -23,7 +23,19 @@
 
 @index_bp.route("/")
 def index():
-    return render_template("index/index.html")
+
+    # get total listen count
+    try:
```

Comment:
I think we ought to add some caching to this -- maybe for 2-3 minutes? I'd hate to have to make this (albeit small) query for loading the home page each time.

Reply:
`get_total_listen_count` already does caching. 

https://github.com/metabrainz/listenbrainz-server/blob/master/listenbrainz/listenstore/influx_listenstore.py#L120

Reply:
Oh right. Grand. :)